colab-notebooks/README.md

```diff
@@ -0,0 +1,8 @@
+# Colabs
+  
+Links to interesting colabs demonstrating use of magenta for creative applications and also documentation for magenta. 
+
+[Mxml Documentation Colab](https://colab.research.google.com/github/tensorflow/magenta-demos/blob/master/colab-notebooks/MusicXML_Document_Structure_Documentation.ipynb) : 
+The goal of this colab notebook is to explain the musicxml storage format for digitally storing music files. The main attributes of musicxml as extracted by one of the magenta music library, `musicxml_parser.py`are explained in detail.  The stucture is shown in the form of a flow chart. 
+
+
```

Comment:
Adding comments here because I can't comment on the ipynb file.

For the copyright header at the top, instead of Infocusp, would you mind switching it to say "The Magenta Authors". You can then add an AUTHORS file to the root directory that includes Infocusp.

Reply:
Done https://github.com/tensorflow/magenta-demos/pull/64/commits/fe1f8016ab7eac0dd05ea73092b84e87e0ca33a6